packages/client-config/src/client-config-types/rest_api_client_config.ts

```diff
@@ -0,0 +1,7 @@
+export type RestApiClientConfig = {
+  aws_cloud_logic_custom: {
```

Comment:
I'm not going to raise a big stink over this, but this seems like a super confusing name for rest api config

Reply:
Agree, but nothing we can do. This is for the Gen1 CLI. For new schema, this name is gone.

Reply:
Ah okay, didn't realize this was for the legacy config